### DISCUSSION TITLE
Display Response Status Code in logging list.

### DISCUSSION THREAD

**Discussion by endruszeit** [mockoon/mockoon#1030] [Ideas]
Hello, I have a feature request/ idea. 

It would be nice to have an option to display the status code of the server response in the list tile of the logging tab. 

It would be much easier to find failing calls, which may require a mock while developing.

![image](https://user-images.githubusercontent.com/22318955/236392121-5b5fc675-36da-42d8-a61d-48f593f01a98.png)

Sincerely

**Comment by 255kb** [comment]
Hi, 

Yes why not. 
I wonder if the timestamp is really needed here. It could be shortened, something like "2min ago" with a full timestamp when hovering. And the status code on the right.

**Reply by endruszeit** [reply]
That would be awesome. Thank you.

**Reply by hauptp** [reply]
Hello,
I would say the timestamp as it is now is quite helpful. For example, if you additionally examine server logs, you can find the related entries faster based on the seconds etc.

**Reply by 255kb** [reply]
@hauptp ok, it make sense :) 

@endruszeit I will open a new issue
